### PR TITLE
added the ability to stretch the table

### LINKED ISSUE #157: Feature(tool settings): support stretch to full-width setting
##Problem
For now table could not be stretched to full width of the display

## Solution
Add tool settings that will allow stretching of the table tool to the full width
Add tool tune that will use api method for stretching

reference: [image tool](https://github.com/editor-js/image/blob/master/src/index.ts) supports this feature

### PR DESCRIPTION
**Pull request by parmisan**
## Problem
[https://github.com/editor-js/table/issues/157](url)
## Solution
- added icons for new tune
- added a tune to stretch the table using the api method
- added tune into toolbox

resolves #157

### CODE REVIEW

**Comment by parmisan** [comment]
## Problem
[https://github.com/editor-js/table/issues/157](url)
## Solution
- added icons for new tune
- added a tune to stretch the table using the api method
- added tune into toolbox

File: src/table.js [hunk 1]
```diff
@@ -172,7 +172,7 @@ export default class Table {
     // Determine the position of the cell in focus
     this.table.addEventListener('focusin', event => this.focusInTableListener(event));
   }
-
+
```
Comment (e11sy) [inline src/table.js:175]: remove this please

File: src/table.js [hunk 2]
```diff
@@ -1008,4 +1016,4 @@ export default class Table {
   destroy() {
     document.removeEventListener('click', this.documentClicked);
   }
-}
\ No newline at end of file
+}
```
Comment (e11sy) [inline src/table.js:1019]: unexpected change

File: vite.config.js [hunk 3]
```diff
@@ -25,6 +25,7 @@ export default {
       usePolling: true, 
     },
   },
+
```
Comment (e11sy) [inline vite.config.js:28]: unexpected change

**Comment by neSpecc** [comment]
Please, attach an issue correctly

File: index.html [hunk 4]
```diff
@@ -42,8 +42,8 @@
   <script src="https://cdn.jsdelivr.net/npm/@editorjs/editorjs@latest"></script>
   <script src="./dist/table.umd.js"></script>
```
Comment (neSpecc) [inline index.html:43]: duplicated import

File: package.json
Comment (neSpecc) [inline package.json:1]: unexpected change

File: src/table.js [hunk 5]
```diff
@@ -813,6 +813,14 @@ export default class Table {
     }
   }
 
+  /**
+   * Changes data stretched value and calls the stretchBlock api method
+   */
+  setStretchSetting() {
+    this.data.stretched = !this.data.stretched;
+    this.api.blocks.stretchBlock(this.api.blocks.getCurrentBlockIndex(), this.data.stretched);
```
Comment (neSpecc) [inline src/table.js:821]: this method is deprecated, use BlockAPI `stretched` setter instead

**Review by neSpecc — COMMENTED** [review]
Please, update the readme as well

File: src/plugin.js [hunk 6]
```diff
@@ -130,6 +131,16 @@ export default class TableBlock {
...
+      }, {
+        label: this.data.stretched ? this.api.i18n.t('Collapse') : this.api.i18n.t('Stretch'),
+        icon: this.data.stretched ? IconCollapse : IconStretch,
+        closeOnActivate: true,
+        toggle: true,
+        onActivate: () => {
+          this.data.stretched = !this.data.stretched;
+          this.block.stretched = this.data.stretched;
+          
+        }
```
Comment (e11sy) [inline src/plugin.js:143]:
```suggestion
          this.block.stretched = this.data.stretched;
        }
```

File: src/table.js (hunk 2 above)
Comment (e11sy) [reply src/table.js:1011]: still actual

File: README.md [hunk 7]
```diff
@@ -64,6 +64,7 @@ var editor = EditorJS({
 | `maxRows`          | `number` | maximum number of rows. `5` by params |
 | `maxCols`          | `number` | maximum number of columns. `5` by params |
 | `withHeadings`     | `boolean` | toggle table headings. `false` by default |
+| `stretched`        | `boolean` | toggle table stretch state. `false` by default |
```
Comment (neSpecc) [inline README.md:67]:
```suggestion
| `stretched`        | `boolean` | whether the table is stretched to fill the full width of the container |
```

File: README.md [hunk 8]
```diff
@@ -72,13 +73,15 @@ This Tool returns `data` in the following format
 | Field          | Type         | Description           |
 | -------------- | ------------ | ----------------------------------------- |
 | `withHeadings` | `boolean`    | Uses the first line as headings |
+| `stretched`    | `boolean`    | table stretch state |
```
Comment (neSpecc) [inline README.md:76]:
```suggestion
| `stretched`    | `boolean`    | whether the table is stretched to fill the full width of the container |
```

**Review by neSpecc — COMMENTED** [review]
also, please update a patch version in package.json

File: src/table.js (hunk 2 above)
Comment (neSpecc) [reply src/table.js:1011]: ignored comment

**Review by neSpecc — COMMENTED** [review]
Update a patch version in package.json

**Review by neSpecc — APPROVED** [review]

**Review by e11sy — APPROVED** [review]

**Comment by neSpecc** [comment]
Thanks, @parmisan

**Pull request merged**